.github/workflows/CI.yml

```diff
@@ -41,3 +41,39 @@ jobs:
         run: npx envinfo
       - name: Build
         run: ./configure && make -j8
+  lint-addon-docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
```

Comment:
I deliberately went for the earliest in support version of Node.js on the basis that it would alert us if any of our tooling involved in the linting would break on it and thus help apply `dont-land-on` labels (with an assumption that its very unlikely that something would break on 12 but not 10).

As for speed the lint checks are completing in less than 2 minutes which is a fraction of the time taken to run the build jobs.

Reply:
If we want to have such guarantees, we could even let it run on both versions?

Reply:
Ideally in the long term when https://github.com/actions/setup-node/issues/26 is addressed we can put an alias in here like `lts` if the concern is that the version here will eventually need to be updated.